bin/run-playbook.bash

```diff
@@ -34,7 +34,7 @@ if [ -z "${CK8S_KUBESPRAY_NO_VENV+x}" ]; then
 fi
 
 log_info "Running kubespray"
-ansible-playbook -i "${config[inventory_file]}" "-e serial=1" "${playbook}" "${@}"
+ansible-playbook -i "${config[inventory_file]}" "-e serial=1" "playbooks/${playbook}" "${@}"
```

Comment:
This change also impacts any documentation that uses this function to run playbooks in this repo (e.g. the ones in the [playbooks folder](https://github.com/elastisys/ck8s-cluster-api/blob/main/bin/terminate.bash#L70)). Those snippets need to change from `./bin/ck8s-kubespray run-playbook <prefix> ../playbooks/<playbook>` to `./bin/ck8s-kubespray run-playbook <prefix> ../../playbooks/<playbook>`
Will you try to change those as well? There is at least two [here](https://github.com/elastisys/compliantkubernetes-kubespray/blob/37f7386c39db2a9c806f935687ea0c42d824fd25/README.md?plain=1#L129). There are also likely more in other documents and repos.

Reply:
Sure !!

Reply:
Did a quick check and the only play such commands exist are in v2.19 or below, so i only changed the command in the readme.